docs/topics_5/proc_reporting-issues.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide_5/master.adoc
+
+[id="proc_reporting-issues_{context}"]
+= Reporting issues
+
+[role="_abstract"]
```

Comment:
What is this? `[role="_abstract"]`

Reply:
Automatically added by newdoc. The text that immediately follows the [role="_abstract"] tag will be be part of the search metadata for Pv2.